ftl/common/args.py

```diff
@@ -87,6 +88,20 @@ def extra_args(parser, opt_list):
                         final image'
             }
         ],
+        'python_version': [
+            '--python-version', {
+                "dest":
```

Comment:
Do the keys and values need to be on separate lines here? It would be easier to follow if they were inlined.

Reply:
done